app/web/src/features/communities/EditCommunityInfoPage.tsx

```diff
@@ -109,6 +111,8 @@ export default function EditCommunityPage() {
       {({ community }) => {
         return community.mainPage?.canEdit ? (
           <>
+            <PageHeader page={community.mainPage} />
+            <CommunityPageSubHeader community={community} />
```

Comment:
```suggestion
            <CommunityPageSubHeader community={community} defaultTab="info" />
```
I actually forgot to do this in the original implementation!